mail-client/mutt/mutt-1.6.1.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="Small but very powerful text-based mail client"
+DESCRIPTION="A text-based email client for Unix-like systems. \
+It was originally written by Michael Elkins in 1995 and released \
+under the GNU General Public License version 2 or any later version.
+
+The Mutt slogan is All mail clients suck. This one just sucks less."
+HOMEPAGE="http://www.mutt.org"
+COPYRIGHT="1996-2014 Michael R. Elkins
+	1996-2002 Brandon Long
+	1997-2009 Thomas Roessler
+	1998-2005 Werner Koch
+	1999-2014 Brendan Cully
+	1999-2002 Tommi Komulainen
+	2000-2004 Edmund Grimley Evans
+	2006-2009 Rocco Rutte
+	2014-2015 Kevin J. McCarthy"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="ftp://ftp.mutt.org/pub/mutt/mutt-$portVersion.tar.gz"
+CHECKSUM_SHA256="98b26cecc6b1713082fc880344fa345c20bd7ded6459abe18c84429c7cf8ed20"
+PATCHES="mutt-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
+
+PROVIDES="
+	mutt$secondaryArchSuffix = $portVersion
+	cmd:mutt$secondaryArchSuffix = $portVersion
+	cmd:flea$secondaryArchSuffix = $portVersion
+	cmd:muttbug$secondaryArchSuffix = $portVersion
+	cmd:pgpewrap$secondaryArchSuffix = $portVersion
+	cmd:pgpring$secondaryArchSuffix = $portVersion
+	cmd:smime_keys$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgpgme$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libgpgme$secondaryArchSuffix
+	lib:libgpg_error$secondaryArch >= 0
+	lib:libgettextpo$secondaryArch
+	lib:libassuan$secondaryArch
+	lib:libintl$secondaryArch
+	lib:libassuan$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	cmd:gpgme_config$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libgpgme$secondaryArchSuffix
+	devel:libgpg_error$secondaryArchSuffix
+	devel:libgettextlib$secondaryArchSuffix
+	devel:libassuan$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:autoreconf
+	cmd:bison
+	cmd:gawk
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
```

Comment:
You need to add a **`cmd:libtoolize$secondaryArchSuffix`** to BUILD_PREREQUIRES. The best place for it is just before **`cmd:make`**.
BTW, the **`cmd:gpgme_config$secondaryArchSuffix`** currently in BUILD_REQUIRES would be better in BUILD_PREREQUIRES, between the **`cmd:gcc$secondaryArchSuffix`** and the **`cmd:libtoolize$secondaryArchSuffix`**.